src/client/search_requests.py

```diff
@@ -386,48 +366,44 @@ class CompareFields:
 
 
 class AssignmentSearchRequest(BaseSearchRequest):
-    """Parameters for searching assignment
+    """Parameters for searching task suites assignments.
 
     Attributes:
-        status: The status of an assigned task suite:
-            * `ACTIVE` — Assigned but not completed.
-            * `SUBMITTED` — Completed but not checked.
-            * `ACCEPTED` — Accepted by the requester.
-            * `REJECTED` — Rejected by the requester.
-            * `SKIPPED` — Skipped by the Toloker.
-            * `EXPIRED` — Time for completing tasks has expired.
```

Comment:
Let's either keep these descriptions like for `AppProjectSearchRequest.scope` as these values are not self-describing or move these descriptions in `Assignment.Status` docstring. I like the second option as this enum is used in multiple classes and values are described every time.